main.py

```diff
@@ -711,6 +717,52 @@ def make_positive_or_negative(self) -> None:
 
             self._positive_or_negative_json["data"].append(data)
 
+    def make_warning_and_phase(self) -> None:
+        # warning_and_phase.jsonを生成する
+        self._warning_and_phase_json = {
+            "data": {},
+            "last_update": self.get_inspections_last_update()
+        }
+
+        warning = 0
+        phase = 0
+
+        positive_or_negative_dict = self.positive_or_negative_json()
+        average_length = len(positive_or_negative_dict['data'])
+        latest_average_patients = positive_or_negative_dict['data'][average_length - 1]['7日間平均陽性数']
```

Comment:
```suggestion
        average_length = len(positive_or_negative_dict["data"])
        latest_average_patients = positive_or_negative_dict["data"][average_length - 1]["7日間平均陽性数"]
```
全部ダブルクオーテーションで統一します(注文多くてごめんなさい...)